src/ctapipe/utils/astro.py

```diff
@@ -4,63 +4,231 @@
 not provided by external packages and/or to satisfy particular needs of
 usage within ctapipe.
 """
-from astropy import units as u
+
+import logging
+from enum import Enum
+from pathlib import Path
+
+import astropy.units as u
 from astropy.coordinates import Angle, SkyCoord
+from astropy.table import Table
+from astropy.time import Time
+
+log = logging.getLogger("main")
 
 __all__ = ["get_bright_stars"]
 
 
-def get_bright_stars(pointing=None, radius=None, magnitude_cut=None):
+CACHE_FILE = Path("~/.psf_stars.ecsv").expanduser()
+
+
+class StarCatalogues(Enum):
+    Yale = "V/50/catalog"  # Yale bright star catalogue
```

Comment:
Change the comment to be above and have the form `#: `, this will be picked up by sphinx for the docs